In [1]:
import numpy as np 

def fred(a,b):
    c= (a+b)*12
    return c
a= 13.2
b=3.9
c=fred(a,b)

print("c = ", c)

c =  205.2


In [2]:
def rule (dA, c):
    return c*dA

In [3]:
def rule2 (A,dA,m,c):
    return c*m*(A**(m-1))*dA

In [4]:
def rule3 (dA,dB):
    dQ=np.sqrt(dA**2 + dB**2)
    return dQ

In [9]:
def rule4(Q,A,dA,m,B,dB,n):
    s=((m*dA/A)**2)+((n*dB/B)**2)
    return Q*np.sqrt(s)

$F{c} = $

In [10]:
#lab data 
mass=.2083
d_mass= .001
F=0.7888683507240091
slope= 10.424241737471668
d_slope=1.1929392256645142
rule4 (F,mass,d_mass,1,slope,d_slope,1)

0.09035666366700558

In [11]:
x= 1.1, 1.3, 1.4, 0.9, 0.95, 1.05
st= np.std(x)
avg= np.average(x)
error= st/np.sqrt(len(x))
print(st)
print(avg)

0.17950549357115014
1.1166666666666667


Calculations of x for standard deviation $\sigma = .18$ and the average$ = 1.116 \pm .07$

#Rule 3



$\delta Q = \sqrt{(\delta A/A)^2 + (\delta B/B)^2}$

$\Sigma F_c = \frac{4(\Pi)^2*m}{slope}$